app/views/insured/families/_enrollment_refactored.html.erb

```diff
@@ -3,164 +3,165 @@
-            </button>
-
-            <button tabindex="0" onkeydown="handleContactInfoKeyDown(event, 'plan_contact_info-<%= hbx_enrollment.hbx_id%>', '<%= product.kind.to_s %>-<%= product.id %>-<%= hbx_enrollment.hbx_id%>')" class="btn btn-default">
-                <a class="vertically-aligned-row" id="plan_contact_info-<%= hbx_enrollment.hbx_id%>" data-toggle="modal" data-target="#<%= product.kind.to_s %>-<%= product.id %>-<%= hbx_enrollment.hbx_id%>">
-                    <i class="fa fa-phone"></i>
-                    <span class="col-xs-11 enrollment-tile-summary"><%= l10n("plan_contact_info") %></span>
-                </a>
-            </button>
-        <%# popup for phone %>
-            <%= render partial: "shared/#{Settings.aca.state_abbreviation.downcase}_carrier_contact_information", locals: { plan: product, hbx_enrollment: hbx_enrollment } unless product.issuer_profile.blank? %>
+        <div class="right-column">
+          <div class="enrollment-premium">
+            <label><%= l10n("premium_you_pay") %>:</label>
+            <%= number_to_currency(current_premium(hbx_enrollment), precision: 2) %>/<%= l10n("month").to_s.downcase %>
+          </div>
 
-            <%= render partial: "insured/families/enrollment_actions", locals: { read_only: read_only, hbx_enrollment: hbx_enrollment }  %>
+          <% if hbx_enrollment.consumer_role.present? && (hbx_enrollment.applied_aptc_amount > 0) %>
+            <div class="aptc-amount">
+              <label><%= l10n("APTC_amount") %>:</label>
+              <%= number_to_currency(hbx_enrollment.applied_aptc_amount, precision: 2) %>/<%= l10n("month").to_s.downcase %>
+            </div>
+          <% end %>
+
+          <div class="plan-metal-level">
+            <label><%= l10n("metal_level") %>:</label>
+            <%= display_dental_metal_level(product) %>
+          </div>
+
+          <div class="plan-type">
+            <label><%= l10n("type") %>:</label>
+            <%= product.try(:product_type).try(:upcase) %>
+          </div>
+
+          <% unless EnrollRegistry.feature_enabled?(:hide_enrollment_market_type) %>
+            <div class="enrollment-type">
+              <label><%= l10n("market_type") %>: </label>
+              <%= show_market_name_by_enrollment hbx_enrollment %>
+            </div>
+          <% end %>
+
+          <% if hbx_enrollment.eligible_child_care_subsidy > 0 %>
+            <div class="hc44cc-discount">
+              <label><%= l10n("hc44cc_premium_discount") %>:</label>
+              <%= number_to_currency(hbx_enrollment.eligible_child_care_subsidy, precision: 2)%>/<%= l10n("month").to_s.downcase %>
+            </div>
+          <% end %>
         </div>
+
+      </div>
+
+      <div class="plan-covered">
+        <% members = covered_members_name_age(hbx_enrollment.hbx_enrollment_members) %>
+        <% if members.present? %>
```

Comment:
any harm in removing this? without it, it'd be slightly more readable. the end result would be looping over members on line 140, and if it's empty, would just create an empty div